src/lib/y2partitioner/actions/controllers/filesystem.rb

```diff
@@ -338,7 +341,12 @@ def finish
           if to_be_encrypted?
             name = Y2Storage::Encryption.dm_name_for(blk_device)
             enc = blk_device.create_encryption(name)
-            enc.password = encrypt_password
+
+            if random_password
+              enc.password_file = '/dev/urandom' # FIXME
```

Comment:
FIXME what? without details it is not much useful

Reply:
Just a note to myself to not forget to review and ideally delete this literal from the code, because it is not clear to me yet that this will be the final way to set the random password for SWAP partition.

Anyway, I am going to remove it or add details, as appropriate, in next update.

Thanks.